internal/telemetry/log/log.go

```diff
@@ -3,52 +3,105 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2025 Datadog, Inc.
 
+// Package log provides secure telemetry logging with strict security controls.
+//
+// SECURITY MODEL:
+//
+// This package implements strict security controls for telemetry logging to prevent
+// PII and sensitive information from being sent to external telemetry services.
```

Comment:
Awesome thanks 👍 

One additional element is that constant messages also allows us to better deduplicate log messages to reduce how many of the same log we actually send